components/widgets/resources/src/main/resources/resources/uicomponents/PhenotypeSelectionUtils.js

```diff
@@ -41,7 +41,7 @@ define("PhenotypeSelectionUtils", [], function() {
            return null;
         }
         var id = phenotype.id;
-        var searchUrl = new XWiki.Document('SolrService', 'PhenoTips').getURL("get", "q=" + id);
+        var searchUrl = XWiki.contextPath + "/rest/vocabularies/hpo/" + id;
```

Comment:
in this entire pull request sometimes a URL like `"/rest/vocabularies/hpo/" + id` is used, and sometimes `"/rest/vocabularies/hpo?input =id`, whats the difference and why\when do we use one and why\when the other?